style.css

```diff
@@ -0,0 +1,3 @@
+.headertext {
+  color: blue;
+}
```

Comment:
[OPTIONAL] It is considered bad practice to use color names. I recommend you change this color to its `rgb` or `hex` equivalent. The `rgb` code for this color is `rgb (0, 0, 255)` and the `hex` code for this color is `#0000ff`

Reply:
Thanks I used the rgb